hedera-node/src/test/java/com/hedera/services/stream/RecordStreamTypeTest.java

```diff
@@ -34,6 +34,7 @@ class RecordStreamTypeTest {
 	private static final String expectedRecordDescription = "records";
 	private static final String expectedRecordExtension = "rcd";
 	private static final String expectedRecordSigExtension = "rcd_sig";
+	private static final String expectedSidecarRecordExtension = "rcd";
```

Comment:
Just use `expectedRecordExtension` no?
Unless you change this to a regex that captures the format

Reply:
True, currently the values are the same --- it was done mainly for semantic purposes and to avoid confusion/ambiguity. Even though the sidecar file extension is currently the same as the record files one, that may change in the future... There is a separate sidecar value as well defined in [RecordStreamType](https://github.com/hashgraph/hedera-services/pull/3625/files/3bdc5c65d9117bcd3a4018992dd041424f7e4c6f#diff-9de5ea09319954ac9a7d023162457a52a91e0c32174ccc5eab484fa2b5601d9fR33-R35). Can remove if deemed unnecessary/not helping with the clarity?